hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -1,12 +1,18 @@
 ---
-- name: Sync repos for controller to compute for periodic jobs
+- name: Sync repos for controller to compute for periodic jobs and gating repo
   hosts: computes
   gather_facts: true
   tasks:
+    - name: Check for gating repo on controller-0
+      delegate_to: controller
```

Comment:
shouldn't it be `controller-0` ? Unless the task name is misleading?

Reply:
The task name and hostname both are correct. Zuul ansible node has controller hostname. If we want to use controller-0. I can try that.

Reply:
context: this is running in CI - `controller-0` doesn't exist, so yep, `controller` it is.